crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/service/EformStagingServiceTest.java

```diff
@@ -25,16 +23,6 @@ class EformStagingServiceTest {
     @InjectMocks
     private EformStagingService eformStagingService;
 
-    @Test
-    void givenValidParamsAndEformStagingUsnCreatedByHubUser_whenEformStagingServiceIsInvoked_thenCrimeApplicationExceptionIsThrown() throws IOException {
-        EformStagingResponse retrievedData = EformStagingResponse.builder().maatRef(1001).usn(6000308).userCreated("HUB").build();
-
-        when(eformStagingApiClient.retrieveOrInsertDummyUsnRecordInEformStaging(anyLong()))
-                .thenReturn(retrievedData);
-
-        assertThrows(CrimeApplicationException.class, invokeEformStagingServiceForRetrieveOrInsertDummyUsnRecord());
-    }
-
     @Test
     void givenValidParamsAndEformStagingUsnNotCreatedByHubUser_whenEformStagingServiceIsInvoked_thenEformStagingRecordIsRetrievedForUsn() throws IOException {
```

Comment:
As there is no check on hub user you can change the name of the test case accordingly?

Reply:
Done

Reply:
Same changes need in Integration and controller test

Reply:
done